gnpy/core/service_sheet.py

```diff
@@ -73,8 +73,14 @@ def __init__(self,Request,eqpt_filename):
             exit()
         # excel input are in GHz and dBm
         self.spacing = Request.spacing * 1e9
-        self.power =  db2lin(Request.power) * 1e-3
-        self.nb_channel = int(Request.nb_channel)
+        if Request.power is not None:
```

Comment:
Can we switch this to `request` with a lowercase `r`?

Reply:
Nevermind. We'll fix this and a couple other superficial things with a follow-up PR.